logging_transport.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
I think this shouldn't be a supported component of the SDK. I suggest you add it to the example.